### DISCUSSION TITLE
Fehler Timeout

### DISCUSSION THREAD

**Discussion by ciddi89** [Zefau/ioBroker.jarvis#943] [Q&A] [answered]
Hallo,

leider bekomme ich bei v3 immer wieder ein Timeout Fehler und komme garnicht in das Interface von Jarvis. 

Was habe ich schon gemacht? 
- Cache geleert
- ioBroker neu gestertet
- in der Konsole "iobroker upload Jarvis"
- zurück auf v2 funktioniert dann wieder alles.

<img width="1708" alt="Bildschirmfoto 2021-09-03 um 09 33 37" src="https://user-images.githubusercontent.com/66088020/131969396-f4f845fa-41af-4a00-94e4-66f8b4ea12e3.png">

Vielen Dank schonmal

LG

**Comment by Zefau** [comment]
Dein Socket-Port ist falsch. Wie ist die ioBroker.jarvis Adapter-Einstellung? Hast du die Verbindungseinstellung auf `auto`?

**Comment by ciddi89** [comment]
> Dein Socket-Port ist falsch. Wie ist die ioBroker.jarvis Adapter-Einstellung? Hast du die Verbindungseinstellung auf `auto`?

Ja steht alles auf Auto. Siehe Screenshot 
<img width="755" alt="Bildschirmfoto 2021-09-03 um 10 29 53" src="https://user-images.githubusercontent.com/66088020/131975551-7902e964-e063-47c4-a9dc-19890e810229.png">

**Comment by ciddi89** [comment]
Anstatt des 404 not found Fehlers bekomme ich nun das in der Browser Konsole ausgegeben wenn ich die Seite Aktualisiere:
<img width="696" alt="Bildschirmfoto 2021-09-03 um 10 34 02" src="https://user-images.githubusercontent.com/66088020/131976120-21da13ac-f2b0-46b1-8ef7-37b4bdd7481e.png">

**Comment by Zefau** [comment, marked as answer]
Dann trag mal `iobroker` in die Whitelist ein (in den Adapter-Einstellungen; rechts neben Eingabefeldern, die auf deinem Screenshot zu sehen sind).

**Comment by ciddi89** [comment]
Wow das war's!  Vielen Dank für die schnelle Hilfe! 🤩 Habe zur Sicherheit neben dem Hostnamen auch die ip Adresse eingegeben. Nun läuft es bei beiden. 👍🏻

**Comment by Haroldmentor** [comment]
Hey, bin neu und habe heute das erste mal verucht Jarvis in IOBroker zu installieren...

Bekomme hier nur leider auch immer ein Timeout:
![image](https://user-images.githubusercontent.com/107301673/173181879-8f7eda4c-9c2d-4fa6-bc98-1a3f3ae80369.png)

Da ich neu bin weiß ich nicht ob es überhaupt hier hin gehört...
Was braucht ihr noch für Infos von mir?

**Reply by mcuiobroker** [reply]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/besonderheiten-v3/fehler-hinweise-v3#error-timeout-oefter-bei-unterschiedlichen-geraeten
Und in die WhiteList iobroker eintragen.

**Reply by Zefau** [reply]
@Haroldmentor hast du es beheben können?

**Reply by Haroldmentor** [reply]
Jein. Wenn ich statt meinen Namen "IOBroker" die IP benutze funtkioniert es...

**Reply by Zefau** [reply]
Wie sieht denn die Whitelist aus, die du in der Adapter Konfiguration drin hast?

**Reply by Haroldmentor** [reply]
Habe ich alle möglichen Einträge durchprobiert ohne das sich was verändert habe. Habe dieses Thema auch in der IOBroker Community diskutiert... Hier habe ich allerlei Tipps bekommen die aber nichts verändert haben...

**Reply by Zefau** [reply]
![image](https://user-images.githubusercontent.com/14368811/176022868-2ec1ad9b-a51e-400d-a785-9511fa8a6b80.png)

Mit `Allow from any` sollte es eigl. funktionieren. Im Debug Log sollte dann stehen, von wo die Anfrage kommt und das diese genehmigt ist.
Die Verbindung ist via `http` bei dir, oder?
